### PR TITLE
[VUMM-653] Modify ExperimentRunTest.java to run in each PR

### PR DESCRIPTION
**Pull request by anandjakhaniya**

### CODE REVIEW

**Review by evgeniilezhnin — APPROVED** [review]

File: backend/src/test/java/ai/verta/modeldb/ExperimentRunTest.java [hunk 1]
```diff
@@ -306,7 +347,22 @@ public CreateExperimentRun getCreateExperimentRunRequestSimple(
         .build();
   }
 
-  public static CreateExperimentRun getCreateExperimentRunRequest(
+  public static CreateExperimentRun getCreateExperimentRunRequestForOtherTests(
```
Comment (jkwatson-verta) [inline backend/src/test/java/ai/verta/modeldb/ExperimentRunTest.java]: Maybe this doesn't belong here? Maybe it belongs in some testing utility class?

Comment (anandjakhaniya) [reply backend/src/test/java/ai/verta/modeldb/ExperimentRunTest.java]: Yeah might be, we should create a utility class for this type of thing. Will update PRs.

**Comment by anandjakhaniya** [comment]
@jkwatson-verta

**Review by jkwatson-verta — APPROVED** [review]

**Comment by anandjakhaniya** [comment]
!rebuild

**Pull request merged**